### DISCUSSION TITLE
Welcome all!

### DISCUSSION THREAD

**Discussion by PHLAK** [DirectoryLister/DirectoryLister#535] [General]
Hello and welcome to the Directory Lister discussion boards! I hope this can become a place of useful community interaction.

Feel free to post your general questions and discussion in the [General](https://github.com/DirectoryLister/DirectoryLister/discussions?discussions_q=category%3AGeneral) channel.

For help with installation, configuration and geneal usage of Directory Lister post your questions in the [Help & Support ](https://github.com/DirectoryLister/DirectoryLister/discussions?discussions_q=category%3A%22Help+%26+Support%22)channel.

Lastly, if you would like to discuss any ideas or feature requests you may have head on over to the [Ideas & Features](https://github.com/DirectoryLister/DirectoryLister/discussions?discussions_q=category%3A%22Ideas+%26+Features%22) channel.

Please remember to be polite and have a good time!

**Comment by ghost** [comment]
hello...

**Reply by PHLAK** [reply]
Hi!

**Comment by marchingon12** [comment]
Good sir, my aqquaintance and I believe that the docs were quite vague with the Installation part. We were confused what you meant with where `.env` should be even though your directorylister.com repo didn't include one. Could you help us out with that? Thanks!

**Reply by PHLAK** [reply]
When it's said to copy `.env.example` to `.env` we mean copy it to the same directory. Thus the `.env` should be in the root of your listed directory alongside the `index.php` and `.env.example` files.

```
some-directory/
├── app/
├── directory-lister.svg
├── .env
├── .env.example
├── index.php
├── LICENSE
└── README.md
```

**Reply by marchingon12** [reply]
Ah I see! Thank you for the clarification. If I may, based on you explanation, I would write: "Create a new `.env` file & copy the content of `.env.example` into the `.env` file. The `.env `file should be in the same directory as `index.php` and `.env.examples`". That would be much clearer. Also the directory graph you showed really helped, so you could also include that. Again, thanks a bunch!
